Inleiding.md

```diff
@@ -134,41 +134,70 @@ De opname in een registratie kent vaak een inwinningsproces om gegevenswaarden o
+er nog geen sprake van een informatiemodel, omdat deze modellen kennis modelleren en zich (nog) 
+niet richten op het modelleren van informatie (data en de betekenis hiervan) .
+
+Het modelleren van informatie gebeurt in een informatiemodel. Hierin wordt aangegeven welke 
+objecten welke kenmerken/eigenschappen hebben en of deze kenmerken/eigenschappen in het 
+beschouwde domein verplicht zijn of optioneel zijn enzovoorts. Het informatiemodel geeft 
+hierbij aan welke informatie wordt geregistreerd of uitgewisseld kan worden en beschrijft 
+ook niet meer dan dat. In MIM val een informatiemodel onder niveau 2 of 3. MIM richt zich 
+primair op niveau 2 en 3. Het informatiemodel is hierbij altijd techniek onafhankelijk.
+
+Het informatiemodel kan vervolgens uitgewerkt worden in verschillende soorten technische 
+schema's (zoals XML of JSON of specifieke invullingen hiervan). In MIM wordt dit niveau 4 
+genoemd en voor MIM is dit niveau 4 buiten scope. Wel staat MIM een (model-gedreven 
+werkwijze voor waarbij niveau 4 gegenereerd kan worden vanuit niveau 2 of 3.
+
+De aangegeven niveaus zijn vooral bedoeld om de scope van MIM duidelijk aan geven, 
+te weten niveau 2 en 3. Hieronder worden de niveaus nader toegelicht.
+
 
 #### Niveau 1: Model van begrippen
-Beschrijft de werkelijkheid binnen het beschouwde domein (de ‘universe of
-discourse’) d.m.v. de daarin gehanteerde begrippen en hun relaties tot elkaar.
-Doel is dat de actoren daarbinnen elkaar begrijpen en één taal spreken. Een
-model van begrippen wordt opgesteld voor gebruik door mensen, met name ‘de
-business’. De begrippen worden beschreven in een formele taal, een vocabulaire.
-Een vocabulaire is geen informatiemodel. Begrippen kunnen in meerdere
-informatiemodellen gebruikt worden.
-
-Ten aanzien van begrippen en informatiemodellen:
-- Van een aantal begrippen wordt informatie bijgehouden, niet van alle.   
-- Voor het bijhouden van informatie wordt een gedetailleerde eenduidige structuur aangebracht.
-- De samenhang tussen informatie is vaak (bewust) beperkter dan de samenhang tussen begrippen.
+Beschrijft de werkelijkheid binnen het beschouwde domein (de ‘universe of discourse’) 
+door middel van de daarin gehanteerde begrippen en hun relaties tot elkaar. Een begrip 
+wordt ook wel een concept genoemd, iets waar mensen aan denken en over praten. 
+Doel is dat de actoren binnen een domein elkaar begrijpen en één taal spreken. 
+Een model van begrippen wordt opgesteld voor gebruik door mensen, met name ‘de business’. 
+De begrippen worden beschreven in een formele taal, een vocabulaire. 
+Denk hier aan een woordenboek, een taxonomie, een model van begrippen waarin de onderlinge 
+samenhang tussen begrippen is aangegeven en aan de modelleren van kennis in algemeenheid 
+- maar denk hierbij niet aan een informatiemodel.
+
+Ten aanzien van begrippen en informatiemodellen en het verschil hiertussen:
+
+- Een begrip beschrijft de betekenis van een woord of term, zoals mensen en experts de 
```

Comment:
>Een begrip beschrijft de betekenis van een woord of term, zoals mensen en experts de 
>werkelijkheid beschouwen en begrijpen (en niet de betekenis en structuur van data)

Voorstel:
Een begrip is de combinatie van een term of woord en een definitie. Begrippen worden door mensen gebruikt om mentaal de werkelijkheid te beschouwen en te begrijpen. Een begrip omvat niet de betekenis en structuur van data.